src/Collectors/UsedTranslationViewCollector.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Larastan\Larastan\Collectors;
+
+use Larastan\Larastan\Support\ViewFileHelper;
+use PhpParser\Node;
+use PHPStan\Parser\Parser;
+use PHPStan\Parser\ParserErrorsException;
+
+use function array_filter;
+use function array_map;
+use function array_merge;
+use function count;
+use function preg_match_all;
+
+use const PREG_SET_ORDER;
+
+final class UsedTranslationViewCollector
+{
+    /** @see https://regex101.com/r/ksUgGz/1 */
+    private const TRANSLATION_REGEX = '/[^\w](trans|trans_choice|Lang::get|Lang::choice|Lang::trans|Lang::transChoice|@lang|@choice|__|\$t)\((?P<quote>[\'"])(?P<string>(?:\\k{quote}|(?!\k{quote}).)*)\k{quote}[\),]/m';
+
+    public function __construct(private Parser $parser, private ViewFileHelper $viewFileHelper)
+    {
+    }
+
+    /** @return array<string, array{0: string, 1: int}[]> */
+    public function getUsedTranslations(): array
+    {
+        $translations = [];
+
+        foreach ($this->viewFileHelper->getRootViewFilePaths() as $viewFile) {
+            try {
+                $parserNodes = $this->parser->parseFile($viewFile);
+
+                $translations[$viewFile] = $this->processNodes($parserNodes);
+            } catch (ParserErrorsException) {
+                continue;
+            }
+        }
```

Comment:
I know we do the similar thing in `UsedViewInAnotherViewCollector`

Could you maybe look into if we can cache the parsed views somehow? Maybe a new class with a property inside? Or a static variable, though I'd rather not.